utils/rpms/daos.spec

```diff
@@ -0,0 +1,227 @@
+# Needed because of the GO binaries
+%undefine _missing_build_ids_terminate_build
+
+%define daoshome %{_exec_prefix}/lib/%{name}
+
+Name:          daos
+Version:       0.4.0
+Release:       2%{?dist}
+Summary:       DAOS Storage Engine
+
+License:       Apache
+URL:           https//github.com/daos-stack/daos
+Source0:       %{name}-%{version}.tar.gz
+Source1:       scons_local-%{version}.tar.gz
+
+BuildRequires: scons
+BuildRequires: cart-devel
+BuildRequires: argobots-devel >= 1.0rc1
+BuildRequires: libpmem-devel, libpmemobj-devel
+BuildRequires: fuse-devel >= 3.4.2
+BuildRequires: protobuf-c-devel
+BuildRequires: spdk-devel, spdk-tools
+BuildRequires: fio < 3.4
+BuildRequires: libisa-l-devel
+BuildRequires: raft-devel
+BuildRequires: mercury-devel
+BuildRequires: openpa-devel
+BuildRequires: libfabric-devel
+BuildRequires: openssl-devel
+BuildRequires: ompi-devel
+BuildRequires: pmix-devel
+BuildRequires: hwloc-devel
+BuildRequires: libevent-devel
+BuildRequires: libyaml-devel
+BuildRequires: libcmocka-devel
+BuildRequires: readline-devel
+%if (0%{?rhel} >= 7)
+BuildRequires:  numactl-devel
+BuildRequires: CUnit-devel
+BuildRequires: golang-bin
+BuildRequires: libipmctl-devel
+%else
+%if (0%{?suse_version} >= 1315)
+BuildRequires:  libnuma-devel
+BuildRequires: cunit-devel
+BuildRequires: go1.10
```

Comment:
this looks like a static version, Will this need to be updated each time go version updates?

Reply:
Yes, this is just the unfortunate way in which SUSE packages up go for SLES 12.  Hopefully they have resolved this in SLES 15.